pkg/formats/influx/influx.go

```diff
@@ -117,7 +91,16 @@ func NewMergedInfluxData(s InfluxDataSet) *InfluxData {
 
 type InfluxDataSet []InfluxData
 
-func (s InfluxDataSet) Bytes() []byte {
+func (f *InfluxFormat) report(err error, format string, args ...interface{}) {
+	str := fmt.Sprintf(format, args...)
+	if !f.invalids[str] {
```

Comment:
There's a small chance this will panic with a concurrent map access. Consider wrapping into a sync.RWMutex?

Reply:
Good point. Will do.